client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content.tsx

```diff
@@ -173,14 +182,20 @@ export default function SiteStatusContent( {
 			/>
 		);
 	}
+	let content;
```

Comment:
Just a nitpick. We can keep this var where it was earlier, after the return statement for stats